src/mudlet-lua/tests/DB_spec.lua

```diff
@@ -371,18 +371,36 @@ describe("Tests DB.lua functions", function()
       assert.are.same(test, res[1])
     end)
 
-    it("should successfully delete columns in a non empty table.",
+    it("should successfully delete columns in a non empty table if force argument is provided.",
     function()
       local test = { name = "foo", id = 500, blubb = "bar" }
       db:add(mydb.sheet, test)
-      mydb = db:create("mydbt_testingonly", { sheet = { name = "", id = 0 }})
+      mydb = db:create("mydbt_testingonly", { sheet = { name = "", id = 0 }}, true)
       local res = db:fetch(mydb.sheet)
       test.blubb = nil -- we expect the blubb gets deleted
       assert.are.equal(1, #res)
       res[1]._row_id = nil --we get the row id back, which we don't need
       assert.are.same(test, res[1])
     end)
-
+		
```

Comment:
the indents are also hit or miss in this section, some 2 space some 4. Typically we use a 2 space indent for the Lua stuff.

Reply:
Tabs now converted to two-spaces.